meetings/2025-05-13-taskforce-meeting.md

```diff
@@ -0,0 +1,87 @@
+- a: What if there are two labels, one says "war" and one says "depiction of war"? Who gets to decide two labels are the same thing? Maybe we have a two-tier system where a label publisher can suggest that their label is a subclass or equivalent label. 
+- Jaz: no one really finds equivalences in that ecosystem. It's more just a choice of who people subscribe to.
+- Emelia: right, we want to avoid string matching any more.
+- Jaz: why don't we just adopt what is on the FIRES link you posted?
+- Emelia: https://github.com/fedimod/fires/issues/68 issue on specifying recommended actions on content
+- Darius: do we need to specify actions as objects?
+- denny:  I don't see why specifying the action is within the ambit of a labeler's job. Wouldn't have occured to me as an application developer to look for action information in the label object. 
+- Emelia: I think you make it a string and we specify a set of starter strings
+- Darius: are we reinventing mime-types here?
+- Emelia: maybe we publish a repository of possible actions?
+- a: we probably don't want to take on too much at the start, so as a group here maybe we focus on representation and discovery and back off of the "what happens to the label". It seems like we have the basic building blocks for representation. For labels that clearly the object reference type system as agreed on by some Actor (Person/Service/etc), and then CW is more of freeform text and explicit warning rather than summary. Maybe we leave actions out of step 1
+- Emelia: I totally agree. I bunted on the idea of suggested actions for FIRES for that reason. I think it's something that can be added at a different point in time. For content warnings, they can just be a string. Given that the server is the publisher of the object, it's fine if moderators can add a CW toa post and not have it come from the end user. Whether you want to distinguish whether it was user vs admin, maybe, but that might defeat the point of a free text CW
+- a: re: the question of who is applying a thing, you can add the attribution if it's a label. What matters is how you discover what applied the label. If it's by the publisher they can just say the attribution. If it's something you can discover, then you need to be able to discover where it came from.
+- (discussion of: who created the label is different from who applied a label somewhere)
+- Darius: so we have talked about Objects a lot but not Activity, so maybe the Activity tells us who is applying a label to an object, but the Object itself is just containing information about who created the label originally
+- Emelia: so maybe we focus on Object scope for a current draft and we leave the Activity scope for the future. I don't think it would be useful for someone to say "I put a label on my own post" to be discovered through a side channel.
```

Comment:
```suggestion
- Darius: So we have talked about Objects a lot, but not Activity, so maybe the Activity tells us who is applying a label to an object, but the Object itself just contains information about who originally created the label?
- Emelia: So maybe we focus on Object scope for a current draft, and we leave the Activity scope for the future. I don't think it would be useful for someone to say, "I put a label on my own post," to be discovered through a side channel.
```